xDrip/Scenes/Entries/EntriesList/EntriesListModels.swift

```diff
@@ -15,13 +15,63 @@ import UIKit
 enum EntriesList {
     // MARK: Models
     
+    enum EntryType {
+        case bolus
+        case carbs
+    }
+    
+    struct CellData {
+        let value: String
+        let date: String
+    }
+    
     // MARK: Use cases
     
     enum Load {
         struct Request {
         }
         
         struct Response {
+            let entries: [AbstractEntry]
+            let type: EntryType
+        }
+        
+        struct ViewModel {
+            let cellData: [CellData]
```

Comment:
What about sections? They might be dynamic.

Create
```swift
struct SectionViewModel {
  let title: String
  let items: [CellViewModel]
}
```

and pass `[SectionViewModel]` instead.